keras_core/optimizers/sgd_test.py

```diff
@@ -21,7 +21,7 @@ def test_config(self):
     def test_single_step(self):
         optimizer = SGD(learning_rate=0.5)
         self.assertEqual(len(optimizer.variables), 2)
-        grads = np.array([1.0, 6.0, 7.0, 2.0])
```

Comment:
Do we need to support `np.array` for `apply_gradients`? @fchollet

Reply:
No we don't.